rv_v

```diff
@@ -138,6 +138,11 @@ vfwnmacc.vf    31..26=0x3d vm vs2 rs1 14..12=0x5 vd 6..0=0x57
 vfwmsac.vf     31..26=0x3e vm vs2 rs1 14..12=0x5 vd 6..0=0x57
 vfwnmsac.vf    31..26=0x3f vm vs2 rs1 14..12=0x5 vd 6..0=0x57
 
+#Pseudo OPFVF
+$pseudo_op rv_v::vmflt.vf    vmfgt.vv 31..26=0x1b vm rs1 vs2 14..12=0x5 vd 6..0=0x57
+$pseudo_op rv_v::vmfle.vf    vmfge.vv 31..26=0x19 vm vs2 rs1 14..12=0x5 vd 6..0=0x57
```

Comment:
How can a `vv` variant be a pseudo of a `vf` variant?  I assume what you want here is something like

```
$pseudo_op rv_v::vmflt.vv    vmfgt.vv 31..26=0x1b vm vs1 vs2 14..12=0x5 vd 6..0=0x57
$pseudo_op rv_v::vmfle.vv    vmfge.vv 31..26=0x19 vm vs1 vs2 14..12=0x5 vd 6..0=0x57
```

but I don't see how this communicates to the downstream tooling that vs1 and vs2 are swapped.  Are we missing some additional functionality that we need to implement before downstream tools can make use of these swapped-argument pseudos?

Reply:
This are wrong, sure, they should be vv, my bad. I'm not quite sure yet how they are parsed differently, I'll take a look and answer back on this.

Reply:
Instruction descriptionts are fixed. I'll wait for #283  to do the rest of the required work.